staging/src/k8s.io/apimachinery/pkg/util/managedfields/extract_test.go

```diff
@@ -115,7 +115,7 @@ func TestExtractInto(t *testing.T) {
 				t.Fatalf("Error accessing object: %v", err)
 			}
 			accessor.SetManagedFields(tc.managedFields)
-			err = ExtractInto(tc.obj, tc.objType, tc.fieldManager, &out)
+			err = ExtractInto(tc.obj, tc.objType, tc.fieldManager, &out, "")
```

Comment:
You want to test this with "status" as well?

Reply:
Yes, adding.

Reply:
Fixed